### DISCUSSION TITLE
关于notion模板

### DISCUSSION THREAD

**Discussion by suselee** [reuixiy/reuixiy#17] [Q&A | 问答] [answered]
您好，看了您的notion页面，很喜欢您这套模板；尝试进行还原总是有些问题无法解决！
以**知识**类下的**哲学**块举例，**哲学**block是page类型还是database类型？是怎样做到icon和标题水平对齐的呢。
因为如果为page类型，那icon只能在标题上方；而转成database类型，标题可以icon对齐，可是database view又是怎么做出来的呢，即怎样引用omnis数据库呢？谢谢

**Comment by reuixiy** [comment, marked as answer]
各区块都只是 Omnis 数据库的视图，只不过每个区块设置不同的 filter 条件，以过滤出该区块的数据。

> 可是database view又是怎么做出来的呢，即怎样引用omnis数据库呢？

刚好当时有记录过 https://reuixiy.notion.site/Notion-8c9933ddc5e949918f0d00a7f0b53f41#0c996501b8324c75bf14e057b9ff28e7

![Arc 2023-03-10 at 22 35 53@2x](https://user-images.githubusercontent.com/28993043/224343273-ba04a69e-db2f-4284-970c-c54eedb5ef6f.png)

**Reply by suselee** [reply]
非常感谢！
